app/views/kaminari/_last_page.html.erb

```diff
@@ -0,0 +1,4 @@
+<span class="last">
+  <%= link_to_unless current_page.last?, t("views.pagination.last").html_safe,
+    url, remote: remote %>
```

Comment:
A suggestion to match the line breaks and sign curly braces in all the files...

Reply:
Done in sha `3c95c89`